src/Tribe/Template.php

```diff
@@ -523,14 +523,18 @@ public function template( $name, $context = array(), $echo = true ) {
 		}
 
 		$name = $template_names[ $cache_name_key ];
+		$origin_folder_appendix = array_diff( $this->folder, $this->template_origin_base_folder );
 
 		if ( ! empty( $this->origin->template_namespace ) ) {
-			$namespace = array_merge( (array) $this->origin->template_namespace, $name );
+			$legacy_namespace = array_merge( (array) $this->origin->template_namespace, $name );
+			$namespace = array_merge( (array) $this->origin->template_namespace, $origin_folder_appendix, $name );
 		} else {
-			$namespace = $name;
+			$legacy_namespace = $name;
+			$namespace = array_merge( $origin_folder_appendix, $legacy_namespace );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 8 spaces but found 1 space

<sub>hash: b13460ff2ee4eeded3f73fe835222367</sub>